ApiReview/API.Polly.Core/NoDocs/Polly/ResilienceContext.cs

```diff
@@ -0,0 +1,25 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Polly.Strategy;
+using Polly.Utils;
+
+namespace Polly;
+
+public sealed class ResilienceContext
+{
+    public CancellationToken CancellationToken { get; set; }
```

Comment:
Does the setter need to be public?

Reply:
Hide setter and only expose on user ask.